app/controllers/users/invitations_controller.rb

```diff
@@ -165,7 +165,10 @@ def invitable_teams
                           .joins(:user_teams)
                           .where('user_teams.role': UserTeam.roles[:admin])
                           .distinct
-                          .select { |team| can_invite_team_users?(team) }
+
+      teams = teams.where('name ILIKE ?', "%#{params[:query]}%") if params[:query].present?
```

Comment:
Why don't you use 'where_attributes_like'?

Reply:
fixed